src/foam/nanos/crunch/lite/CapabilityRefinement.js

```diff
@@ -27,7 +27,7 @@ foam.CLASS({
         CrunchService crunchService = (CrunchService) x.get("crunchService");
         List<String> prereqCapIds = crunchService.getPrereqs(x, getId(), null);
 
-        if ( prereqCapIds == null || prereqCapIds.size() == 0 ) return payload.getStatus();
+        if ( prereqCapIds == null || prereqCapIds.size() == 0 ) return GRANTED;
```

Comment:
Is this right? Is there a way to try granting this rather than just returning GRANTED? payload.getStatus() defaults to Action required which never resolves if there are no prereqs

Reply:
this seems correct based off the calling code

Reply:
Note, the current payload still needs to pass validation to the granted. If the payload data is incorrect it still goes into action required in the SetCapablePayloadStatusOnPut rule